docs/src/main/sphinx/connector/jdbc-procedures.fragment

```diff
@@ -9,3 +9,14 @@
   USE example.example_schema;
   CALL system.flush_metadata_cache();
   ```
+
+- `system.update('query')`
+
+  The `update` procedure allows you to query the underlying database directly.
+  For example, the following system call drops the default value from `your_column`
+  on `your_table` table in the `example` catalog
+
+  ```sql
```

Comment:
> The `update` procedure allows you to query the underlying database directly.

Can you pls document whether there is any "result" to such a procedure call ?

Also from end user perspective - it is not clear when to use `query` pass-through and when to use `update`.